jax/lax_linalg.py

```diff
@@ -316,3 +323,49 @@ def qr_jvp_rule(primals, tangents, full_matrices):
 qr_p.def_abstract_eval(qr_abstract_eval)
 xla.translations[qr_p] = qr_translation_rule
 ad.primitive_jvps[qr_p] = qr_jvp_rule
+
+
+# SVD decomposition
```

Comment:
Nit: "decomposition" is redundant, perhaps just expand the acronym ("Singular Value Decomposition")

Reply:
Oh, my bad. Will fix.